synapse/handlers/appservice.py

```diff
@@ -199,8 +202,9 @@ def notify_interested_services_ephemeral(
         Args:
             stream_key: The stream the event came from.
 
-                `stream_key` can be "typing_key", "receipt_key" or "presence_key". Any other
-                value for `stream_key` will cause this function to return early.
+                `stream_key` can be "typing_key", "receipt_key", "presence_key" or
+                "to_device_key". Any other value for `stream_key` will cause this function
+                to return early.
```

Comment:
> Any other value for `stream_key` will cause this function to return early.

will it? it looks like you've removed the relevant return statement. (which seems a surprising choice, but eh)